src/base-api-client.ts

```diff
@@ -0,0 +1,90 @@
+import { ApiRequestHandler } from './api-request-handler';
+
+type ServiceApi = 'topic_extraction' | 'sentiment_analysis';
```

Comment:
I wonder if this is useful, I don't really see the point. If we're going to have it we should have one for version as well.